packages/docusaurus-1.x/lib/core/Head.js

```diff
@@ -172,6 +172,7 @@ class Head extends React.Component {
             }}
           />
         )}
+        <script src="/js/scrollSpy.js" />
```

Comment:
This assumes a baseUrl of /, which is an incorrect assumption. See L186 where a similar script was added for the codetabs functionality.

Reply:
Done